cms/settings.py

```diff
@@ -31,7 +31,8 @@
     "markdown_deux",
 ]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
+    "django.middleware.security.SecurityMiddleware",
```

Comment:
Why do we need this new middleware?

Reply:
I'll take it out

Reply:
Actually sorry please leave this in.

We do need to define these headers ourselves, as we're doing [here](https://github.com/canonical/canonicalwebteam.flask-base/blob/main/canonicalwebteam/flask_base/app.py#L25) for Flask sites. But this isn't Flask, so we'll have to replicate these behaviours in here, and this middleware seems like probably the right way to do it.

Reply:
Done!